Source/WebKit/UIProcess/API/C/WKPage.cpp

```diff
@@ -1314,10 +1314,13 @@ void WKPageSetPagePolicyClient(WKPageRef pageRef, const WKPagePolicyClientBase*
         explicit PolicyClient(const WKPagePolicyClientBase* client)
         {
             initialize(client);
+
+            // This callback is unused and deprecated.
+            RELEASE_ASSERT(!m_client.unableToImplementPolicy);
```

Comment:
This seems a bit aggressive.

Reply:
This is done elsewhere in this file to ensure that if someone is using that SPI, we will notice instead of just having silent failures that are hard to notice.